cla-frontend-contributor-console/.serverless_plugins/serverless-lambda-version/.eslintrc

```diff
@@ -1,4 +1,7 @@
 ---
+# Copyright The Linux Foundation and each contributor to CommunityBridge.
+# SPDX-License-Identifier: MIT
+
```

Comment:
The files in serverless-lambda-version are from a third-party upstream and are not original to CommunityBridge, correct? If so then these notices should _not_ be added to those files.